api-common-java/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +0,0 @@
-Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
```

Comment:
Should we have a PR template referencing `CONTRIBUTING.md`?

Reply:
Added, with a reference to `CONTRIBUTING.md` at the root.